pkg/webhooks/server.go

```diff
@@ -124,19 +129,21 @@ func (ws *WebhookServer) Stop() {
 
 // HandleMutation handles mutating webhook admission request
 func (ws *WebhookServer) HandleMutation(request *v1beta1.AdmissionRequest) *v1beta1.AdmissionResponse {
-	glog.Infof("Handling mutation for Kind=%s, Namespace=%s Name=%s UID=%s patchOperation=%s",
-		request.Kind.Kind, request.Namespace, request.Name, request.UID, request.Operation)
 
 	policies, err := ws.policyLister.List(labels.NewSelector())
 	if err != nil {
 		glog.Warning(err)
 		return nil
 	}
+	glog.V(3).Infof("Handling mutation for Kind=%s, Namespace=%s Name=%s UID=%s patchOperation=%s",
```

Comment:
Should we move this after line 146, we may still need this info in the log to give us the details about admission request.

Reply:
Done